datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/system/kafka/KafkaSetup.java

```diff
@@ -0,0 +1,38 @@
+package com.linkedin.datahub.upgrade.system.kafka;
+
+import com.google.common.collect.ImmutableList;
+import com.linkedin.datahub.upgrade.UpgradeStep;
+import com.linkedin.datahub.upgrade.system.BlockingSystemUpgrade;
+import com.linkedin.metadata.config.kafka.KafkaConfiguration;
+import io.datahubproject.metadata.context.OperationContext;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+
+@Slf4j
+public class KafkaSetup implements BlockingSystemUpgrade {
+
+  private final List<UpgradeStep> _steps;
+
+  public KafkaSetup(
+      OperationContext opContext,
+      KafkaConfiguration kafkaConfiguration,
+      KafkaProperties kafkaProperties) {
+    _steps =
+        ImmutableList.of(
+            new WaitForKafkaReadyStep(opContext, kafkaConfiguration, kafkaProperties),
+            new CreateKafkaTopicsStep(opContext, kafkaConfiguration, kafkaProperties),
+            new ConfluentSchemaRegistryCleanupPolicyStep(
```

Comment:
Not a blocker, but I think the Cleanup is probably a non-blocking step. We probably didn't make the elasticsearch index step a non-blocking step either though.